validations.py

```diff
@@ -0,0 +1,180 @@
+import argparse
+import glob
+import re
+import subprocess
+import yaml
+
+
+def get_fhir_resource_files(base_dir, recently_changed):
+    if recently_changed is True:
+        git_call = subprocess.run(['git', 'diff', '--name-only', 'master...'], check=True, capture_output=True)
+        grep_call = subprocess.run(['grep', base_dir], input=git_call.stdout, capture_output=True)
+        unique_call = subprocess.run(['uniq'], input=grep_call.stdout, capture_output=True)
+        output = unique_call.stdout.decode('utf-8')
+        if len(output) == 0:
+            return []
+        return output.strip().split('\n')
+    else:
+        return glob.glob(f'{base_dir}/*.json')
+
+
+def any_filters_match(patterns, error):
+    return any(re.match(regex, error) for regex in patterns)
+
+
+def get_global_filter(ignore_list):
+    global_filters = []
+
+    if type(ignore_list) is dict:
+        if 'global_filter' in ignore_list and type(ignore_list['global_filter']) is dict:
+            global_filter = ignore_list['global_filter']
+
+            if 'error_patterns' in global_filter and type(global_filter['error_patterns']) is list:
+                global_filters = global_filter['error_patterns']
+
+    return global_filters
+
+
+def get_file_filter(ignore_list, file_path):
```

Comment:
They are logically two separate concepts with two separate code paths.  If there was a common method put in front, it would still make sense to break the logic up into two separate sub methods.  Also, the global filters are only needing to be parsed once, the file filters are parsed for each file, so combining them, you'd duplicate the work of parsing the global filters for each file, which is unnecessary.